internal/oci/runtime_vm.go

```diff
@@ -657,17 +667,75 @@ func (r *runtimeVM) UpdateContainerStatus(ctx context.Context, c *Container) err
 	return r.updateContainerStatus(ctx, c)
 }
 
+func (r *runtimeVM) restoreContainerIO(ctx context.Context, c *Container, state *task.StateResponse) error {
```

Comment:
same here about defining below being called

Reply:
Do you mean put this function after `updateContainerStatus`?

Reply:
yeah! and put `createContainerIO` below `restoreContainerIO` as well (as it calls `createContainerIO`)

Reply:
done